src/languages/index.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const path = require('path');
+
+const translationsPath = path.join(__dirname, 'translations');
+
+function loadTranslations(lang) {
+  try {
+    const filePath = path.join(translationsPath, `${lang}.json`);
+    if (!fs.existsSync(filePath)) {
+      throw new Error(`Translation file for language '${lang}' not found.`);
+    }
+    return JSON.parse(fs.readFileSync(filePath, 'utf-8'));
+  } catch (error) {
+    console.error(error.message);
+    return null;
+  }
+}
+
+function translateLanguage(lang, key) {
+  const translations = loadTranslations(lang);
```

Comment:
Importing the translations every time translateLanguage is called could be slower than importing it just the first time, what do you think saving them in any place before return, so you can re-use the translations imported instead of make a readFileSync each time.

Here is an example:

```suggestion
const translationsPath = path.join(__dirname, 'translations');
const translationsCache = {
   es: {},
   en: {}
};

function loadTranslations(lang) {
  try {
    if (Object.values(translationsCache[lang]).length) return translationsCache[lang]
  
    const filePath = path.join(translationsPath, `${lang}.json`);
    if (!fs.existsSync(filePath)) {
      throw new Error(`Translation file for language '${lang}' not found.`);
    }
    
    const translations = JSON.parse(fs.readFileSync(filePath, 'utf-8'));
    
    return translations
  } catch (error) {
    console.error(error.message);
    return null;
  }
}

function translateLanguage(lang, key) {
  const translations = loadTranslations(lang);
```

Reply:
Completed